autophone.py

```diff
@@ -915,7 +915,9 @@ def on_build(self, msg):
             logger.debug('PULSE BUILD FOUND %s' % msg)
             build_url = msg['packageUrl']
             if msg['branch'] != 'try':
-                tests = PhoneTest.match(build_url=build_url)
+                tests = PhoneTest.match(build_url=build_url,
+                                        branch=msg['branch'],
+                                        revision=msg['revision'])
```

Comment:
build_url is going away since it no longer contains meta data in taskcluster build urls.
should have been repo=msg['branch'] ?
not sure about passing the revision and looking up the changeset inside of PhoneTest.match. More comments later.